src/lib/connect/ws.ts

```diff
@@ -166,11 +169,15 @@ const browserStreamBuilder: StreamBuilder = (client, opts) => {
 
 	const coerceToBuffer = !opts.objectMode
 
+	// the websocket connection
 	const socket = createBrowserWebSocket(client, opts)
+
+	// the proxy is a transform stream that forwards data to the socket
+	// it ensures that data write to socket is a buffer
 	const proxy = buildProxy(opts, socketWriteBrowser, socketEndBrowser)
 
 	if (!opts.objectMode) {
-		proxy._writev = writev
+		proxy._writev = writev.bind(proxy)
```

Comment:
Wondering how this could have work before without the `bind`, maybe Transform automatically handles the bind the `_writev` is set?